client/landing/subscriptions/site-list/site-list.tsx

```diff
@@ -23,9 +23,7 @@ export default function SiteList( { sites }: SiteProps ) {
 					{ translate( 'Email frequency' ) }
 				</span>
 			</li>
-			{ sites.map( ( site ) => (
-				<SiteRow key={ site.id } { ...site } />
-			) ) }
+			{ sites && sites.map( ( site ) => <SiteRow key={ site.ID } { ...site } /> ) }
```

Comment:
Would you say it is alright to use the subscription ID here? 🤔

Reply:
I'd say it's okay as the ID is unique, and you probably won't render anything else with this as a key on this page. But, what I always do is make it a compound value, just to be sure. Like for example `key={ `sites.siterow.${ site.ID }` }`

Reply:
I see. Thank you for the suggestion, Tim! I will apply the change.

Reply:
This is now resolved in https://github.com/Automattic/wp-calypso/pull/75115/commits/76cedbe34f991ee7996eb5174f967c28e83fa120. I have also removed the `key` prop from the body of the `SiteRow` component as I believe it is redundant there.